pkgs/tools/misc/dvtm/default.nix

```diff
@@ -1,20 +1,28 @@
-{ stdenv, fetchurl, ncurses }:
+{ stdenv, fetchurl, ncurses, customConfig ? null }:
 
 stdenv.mkDerivation rec {
+
   name = "dvtm-0.15";
 
-  meta = {
+  meta = with stdenv.lib; {
     description = "Dynamic virtual terminal manager";
     homepage = http://www.brain-dump.org/projects/dvtm;
-    license = stdenv.lib.licenses.mit;
-    platfroms = stdenv.lib.platforms.linux;
+    license = licenses.mit;
+    platfroms = platforms.linux;
+    maintainers = [ maintainers.vrthra ];
   };
 
   src = fetchurl {
     url = "${meta.homepage}/${name}.tar.gz";
     sha256 = "0475w514b7i3gxk6khy8pfj2gx9l7lv2pwacmq92zn1abv01a84g";
   };
 
+  patchPhase = stdenv.lib.optional (customConfig != null) ''
```

Comment:
@joachifm updated. I have a question. I used the `customConfig` originally as a string so that the user can pass in relative paths from where ever they are calling. By passing in the file name as a string, that convenience is no longer there. Do you think it might be better to make customConfig a string again rather than a file name, so that users can do `customConfig = readFile ./dvtm.config.h` rather than `customConfig = "/my/absolute/path/dvtm.config.h"`